aries_vcx/src/protocols/proof_presentation/verifier/verification_status.rs

```diff
@@ -0,0 +1,68 @@
+use serde::{Deserialize, Deserializer};
+
+#[derive(Serialize, Debug, Clone, PartialEq, Eq)]
+pub enum PresentationVerificationStatus {
+    Valid,
+    Invalid,
+    Unavailable,
+}
+
+// todo: to be removed in 0.54.0, this supports legacy serialization when the enum had values "Revoked" and "NotRevoked"
+impl<'de> Deserialize<'de> for PresentationVerificationStatus {
+    fn deserialize<D>(deserializer: D) -> Result<Self, D::Error>
+    where
+        D: Deserializer<'de>,
+    {
+        let s: String = Deserialize::deserialize(deserializer)?;
```

Comment:
Also, you can alias variants directly, like: `#[serde(alias = "NonRevoked")]`. This will attempt deserialization from the variant name as well as the alias/es. Would be shorter than writing the `Deserialize` impl.

Reply:
Are you sure about `#[serde(alias = "NonRevoked")]`? Also, how would it know it should map `NonRevoked` to `Invalid` variant and `Revoked` as `Valid` variant?

Reply:
I've address the `String/&str` comment though

Reply:
The attribute is added to each necessary variant, not on the enum itself

Reply:
Ah right, obivously :-) Thanks, updated